README.md

```diff
@@ -44,22 +67,46 @@ Run this command in the root of the project directory to install NimbleTemplate.
 mix do deps.get, deps.compile
 ```
 
-## Usage
+After the NimbleTemplate is installed, you can continue with the Usage section below.
 
-```bash
-mix help nimble_template.gen # Print help
+## Usage
 
-mix nimble_template.gen -v # Print the version
+Base on the project type, we could choose to run one of the command below for generating the appropriate code structure.
 
+### Phoenix application
+```bash
 # Phoenix application
 mix nimble_template.gen --web   # Apply the Web template
 mix nimble_template.gen --api   # Apply the API template
 mix nimble_template.gen --live  # Apply the LiveView template
-
-# Non-Phoenix application
+```
+### Non-Phoenix application
+```bash
 mix nimble_template.gen --mix # Apply the Mix template
 ```
 
+### Other commands:
+```bash
+mix help nimble_template.gen # Print help
+
+mix nimble_template.gen -v # Print the version
+```
+
+Start up your local server and verify project is ready.
+
+```bash
+mix phx.server
+```
+
+*Note:* 
+
+After the source code initialized successfully by the NimbleTemplate, please remove `nimble_template` in the `mix.exs` 
```

Comment:
Should we change `source code` to `project structure` or we could rephrase to something like

```
Once the NimbleTemplate has been applied, you could remove it out from your mix dependency (`mix.exs`).

Step 1: Remove `nimble_template` dependency in `mix.exs`:

Step 2: Run the following command:
```bash
mix deps.clean --unused --unlock 
```